graylog-storage-elasticsearch7/src/main/java/org/graylog/storage/elasticsearch7/views/searchtypes/pivot/buckets/ESValuesHandler.java

```diff
@@ -105,11 +105,26 @@ private TermsAggregationBuilder createScriptedTerms(List<? extends BucketSpec> b
 
     private Script scriptForPivots(Collection<? extends BucketSpec> pivots) {
         final String scriptSource = Joiner.on(KEY_SEPARATOR_PHRASE).join(pivots.stream()
-                .map(bucket -> "String.valueOf(doc['" + bucket.field() + "'].size() == 0 ? \"N/A\" : doc['" + bucket.field() + "'].value)")
+                .map(bucket -> "String.valueOf((" + documentHasField(bucket.field())
```

Comment:
What about a multiline string interpolation? Wouldn't it increase readability?

Reply:
Good idea. Did that!